server.js

```diff
@@ -68,4 +68,5 @@ app.use(app.router);
 http.createServer(app).listen(port, function(){
   console.log("Lens running on port " + port);
   console.log("http://127.0.0.1:"+port+"/");
+  console.log("To open a local example file goto http://127.0.0.1:"+port+"?url=data/ams/bproc10/bproc10.xml");
```

Comment:
Both of these are specific to a local setup. The latter also carries non-public information.